docs/concepts/index.rst

```diff
@@ -0,0 +1,263 @@
+
+      Objective function
+         A mathematical expression of the energy of a system as a function of
+         binary variables representing the qubits.
+
+      Pegasus
+         The D-Wave :term:`QPU` is a lattice of interconnected qubits. While some qubits
+         connect to others via couplers, the D-Wave QPU is not fully connected.
+         Instead, the qubits interconnect in an architecture known as Pegasus.
+         See a fuller description under :doc:`QPU Topology </concepts/topology>`.
+
+      Penalty function
+         An algorithm for solving constrained optimization problems. In the context
+         of Ocean tools, penalty functions are typically employed to increase the energy
+         level of a problem’s :term:`objective function` by penalizing non-valid configurations.
+         See `Penalty method on Wikipedia <https://en.wikipedia.org/wiki/Penalty_method>`_
+
+      QPU
+         Quantum processing unit
+
+      QUBO
+         Quadratic unconstrained binary optimization.
+         QUBO problems are traditionally used in computer science. Variables
+         are TRUE and FALSE, states that correspond to 1 and 0 values.
+         A QUBO problem is defined using an upper-diagonal matrix :math:`Q`,
+         which is an :math:`N` x :math:`N` upper-triangular matrix of real weights,
+         and :math:`x`, a vector of binary variables, as minimizing the function
+
+         .. math::
+            :nowrap:
+
+            \begin{equation}
+              f(x) = \sum_{i} {Q_{i,i}}{x_i} + \sum_{i<j} {Q_{i,j}}{x_i}{x_j}
+            \end{equation}
+
+         where the diagonal terms :math:`Q_{i,i}` are the linear coefficients and
+         the nonzero off-diagonal terms are the quadratic coefficients
+         :math:`Q_{i,j}`.
+         This can be expressed more concisely as
+
+         .. math::
+            :nowrap:
+
+            \begin{equation}
+              \min_{{x} \in {\{0,1\}^n}} {x}^{T} {Q}{x}.
+            \end{equation}
+
+         In scalar notation, the :term:`objective function` expressed as a QUBO
+         is as follows:
+
+         .. math::
+            :nowrap:
+
+            \begin{equation}
+          		\text{E}_{qubo}(a_i, b_{i,j}; q_i) = \sum_{i} a_i q_i + \sum_{i<j} b_{i,j} q_i q_j.
+            \end{equation}
+
+      Sampler
+         Samplers are processes that sample from low energy states of a problem's objective
+         function, which is a mathematical expression of the energy of a system. A binary
+         quadratic model (BQM) sampler samples from low energy states in models such as those
+         defined by an :term:`Ising` equation or a :term:`QUBO` problem and returns an iterable
+         of samples, in order of increasing energy.
+
+      SAPI
+         Solver API used by clients to communicate with a :term:`solver`.
+
+      Solver
+         A resource that runs a problem. Some solvers interface to the :term:`QPU`;
+         others leverage CPU and GPU resources.
+
+      Source
```

Comment:
Maybe `Source Graph` and `Target Graph`, source and target are used more generally in some places in Ocean I think

Reply:
```
Source
Source graph
```
And same for target